core/AutocorrectSuggestion.h

```diff
@@ -13,13 +13,18 @@ struct AutocorrectSuggestion {
         std::string replacement;
     };
 
-    const std::string title;
+    std::string title;
     std::vector<Edit> edits;
 
     bool isDidYouMean;
 
-    AutocorrectSuggestion(std::string title, std::vector<Edit> edits, bool isDidYouMean = false)
-        : title(title), edits(edits), isDidYouMean(isDidYouMean) {}
+    bool skipWhenAggregated;
+
+    AutocorrectSuggestion() : title(""), edits({}), isDidYouMean(false), skipWhenAggregated(false) {}
```

Comment:
Minor nit: I would be inclined to declare the (`bool`-only) fields as `bool isDidYouMean = false;` in the class definition and then this constructor could just be `AutocorrectSuggestion() = default;`.